en-US/mozorg/newsletters.lang

```diff
@@ -19,6 +20,15 @@ Select country
 Country:
 
 
+## TAG: mozilla_newsletter_2017
+;Subscribe to updates and keep current with Mozilla news. It’s the perfect way for us to keep in touch!
+Subscribe to updates and keep current with Mozilla news. It’s the perfect way for us to keep in touch!
+
+
+;Subscribe to monthly updates and keep current with Mozilla news, including the latest tips and tricks for getting the most out of your Firefox browser. It’s the perfect way for us to keep in touch!
```

Comment:
Interesting, this string is in /newsletter.lang

Let's not add the old string here, only the new one.